charts/fybrik/values.yaml

```diff
@@ -13,9 +13,10 @@ fullnameOverride: ""
 
 clusterScoped: true
 
+# If we work with "clusterScope:false", the namespace must already exist
```

Comment:
I am not sure about this requirement

Reply:
it is required. We have started the discussion today from it.